web-elements/src/test/java/com/demoqa/ButtonsTest.java

```diff
@@ -0,0 +1,127 @@
+package com.demoqa;
+
+import com.demoqa.page.ButtonsPage;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class ButtonsTest extends BaseTest {
+    @Test
+    public void testPageIsOpenTest(){
+        ButtonsPage buttonsPage=new ButtonsPage(driver);
+        buttonsPage.open("buttons");
```

Comment:
No need to provide a parameter  'button'
In BasePage can be PROTECTED method open:
protected void open(String path){
    driver.open(Base_URL + path);
}

and one PUBLIC abstract method open without parameters:
public abstract void open();

Inside each class that extends BasePage you can implement method open:
as the example in ButtonPage
public void open(){
    open("button");
}